### PR TITLE
header, documentation

### PR DESCRIPTION
**Pull request by Dorbedo**
**When merged this pull request will:**
- changes the function header into an ace-like style
- adds a documentation function

### CODE REVIEW

**Comment by Dorbedo** [comment]
The export looks similar like the current wiki:
image

File: addons/ai_hearing/functions/fnc_onSettingChanged.sqf [hunk 1]
```diff
@@ -1,24 +1,23 @@
...
-        BOOL: enable the setting
-
-    Returns:
-        NOTHING
-
-    Example:
-        true call TFAR_ai_hearing_fnc_onSettingChanged;
-*/
+ * Name: TFAR_ai_hearing_fnc_onSettingChanged
+ *
```
Comment (dedmen) [inline addons/ai_hearing/functions/fnc_onSettingChanged.sqf]: Are these * on every line really needed? IMO that's annoying

Comment (Dorbedo) [reply addons/ai_hearing/functions/fnc_onSettingChanged.sqf]: Currently, yes.

**Comment by Dorbedo** [comment]
If you can live with the header, you could merge it.

**Comment by Dorbedo** [comment]
Next Version finished. Now the headers are withozt the `*`.
Ready to merge.

**Comment by Dorbedo** [comment]
Ready again after the header update from ace ;-)

**Comment by dedmen** [comment]
Still ready? Would merge then.

**Comment by Dorbedo** [comment]
jup

**Pull request merged**